sdk/openai/Azure.AI.OpenAI/src/Custom/AudioTranscriptionOptions.Serialization.cs

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#nullable disable
+
+using System.Net.Http;
+using Azure.Core;
+
+namespace Azure.AI.OpenAI
+{
+    public partial class AudioTranscriptionOptions
+    {
+        internal virtual RequestContent ToRequestContent()
+        {
+            var content = new MultipartFormDataRequestContent();
+            content.Add(new StringContent(InternalNonAzureModelName), "model");
+            content.Add(new ByteArrayContent(AudioData.ToArray()), "file", "@placeholderFile.wav");
```

Comment:
Something just occurred to me: Is it an issue if the uploaded file is not a .wav? The API reference says the following formats are supported: flac, mp3, mp4, mpeg, mpga, m4a, ogg, wav, or webm. I would like to believe that the service doesn't take a dependency on the file extension, but it would be good to double-check just to be sure.

Reply:
I was curious about that, too, and I _did_ manually validate three of the supported formats (.wav, .mp3, .ogg). The mismatch between the container filename and actual container didn't matter and it worked great.

It'd be nice to have that represented in the tests somehow, but it's tough since it's a pure service correctness thing.